types/puppeteer/index.d.ts

```diff
@@ -637,6 +644,18 @@ export interface Request {
   /** Contains the request's post body, if any. */
   postData(): string | undefined;
 
+  /**
+   * A `redirectChain` is a chain of requests initiated to fetch a resource.
+   *
+   * - If there are no redirects and the request was successful, the chain will be empty.
+   * - If a server responds with at least a single redirect, then the chain will contain all the requests that were redirected.
+   *
+   * `redirectChain` is shared between all the requests of the same chain.
+   *
+   * @since 1.2.0
+   */
+  redirectChain(): Promise<Request[]>;
```

Comment:
The docs say that `requestChain()` returns no `Promise`. Not sure if the docs are just incorrect though.

https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#requestredirectchain

Reply:
I will make a change.